### PR TITLE
GH-1607: Remove Ghostery rewards toggle, and add language for Cliqz Browser

### PR DESCRIPTION
**Pull request by benstrumeyer**
* [x] Have you followed the guidelines in [CONTRIBUTING.md](../CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you added an explanation of what your changes do?
* [x] Does your submission pass tests?
* [x] Did you lint your code prior to submission?

- Remove Ghostery Rewards feature listing and toggle on Cliqz Browser
- Turn off, gray out, and make non-interactive the Anti-Tracking and Ad Blocking feature listings & toggles 
- Replace the descriptive text beneath Anti-Tracking and Ad Blocking with "This feature is already active in Cliqz by default. Learn more."

### CODE REVIEW

File: app/shared-components/ToggleSwitch/ToggleSwitch.jsx [hunk 1]
```diff
@@ -36,6 +43,8 @@ const ToggleSwitch = (props) => {
 // PropTypes ensure we pass required props of the correct type
 ToggleSwitch.propTypes = {
 	checked: PropTypes.bool.isRequired,
+	locked: PropTypes.bool,
+	removed: PropTypes.bool,
```
Comment (jsignanini) [inline app/shared-components/ToggleSwitch/ToggleSwitch.jsx]: If props aren't marked as `.isRequired`, we should add defaults.

File: app/shared-components/ToggleSwitch/ToggleSwitch.jsx [hunk 2]
```diff
@@ -23,8 +23,15 @@ import ClassNames from 'classnames';
...
 		'ToggleSwitch--active': props.checked,
+		'ToggleSwitch--locked': props.locked,
+		'ToggleSwitch--removed': props.removed
 	});
 
+	ToggleSwitch.defaultProps = {
+		locked: false,
+		removed: false
+	};
+
```
Comment (jsignanini) [inline app/shared-components/ToggleSwitch/ToggleSwitch.jsx]: We add the `.defaultProps` at the bottom of the file, separate from the class or function declaration. See other `.defaultProps` declarations as guidance.

File: app/hub/Views/SetupViews/SetupAntiSuiteView/SetupAntiSuiteViewContainer.jsx [hunk 3]
```diff
@@ -94,44 +97,49 @@ class SetupAntiSuiteViewContainer extends Component {
...
-					this._handleToggle('smart-blocking');
-				},
+				toggle: () => this._handleToggle('smart-blocking'),
 				description: t('hub_setup_smartblocking_description_smartblocking'),
-			},
-			{
+			}
+		];
+
+		if (ghostery_rewards_enabled) {
```
Comment (jsignanini) [inline app/hub/Views/SetupViews/SetupAntiSuiteView/SetupAntiSuiteViewContainer.jsx]: It looks like we're only using `ghostery_rewards_enabled` right here, would it be clearer to replace this with `!IS_CLIQZ` instead?

File: app/shared-components/ToggleSwitch/ToggleSwitch.jsx (hunk 2 above)
Comment (benstrumeyer) [reply app/shared-components/ToggleSwitch/ToggleSwitch.jsx]: Very neat. I have moved it to the bottom of the file.

File: app/hub/Views/SetupViews/SetupAntiSuiteView/SetupAntiSuiteViewContainer.jsx (hunk 3 above)
Comment (benstrumeyer) [reply app/hub/Views/SetupViews/SetupAntiSuiteView/SetupAntiSuiteViewContainer.jsx]: Definitely agree. I've removed the variable entirely and replaced it with !IS_CLIQZ.

**Pull request merged**